rpa-manager/modules/ROOT/pages/processoperations-deploymentmap.adoc

```diff
@@ -1,11 +1,19 @@
 = Deployment Map
 
-The deployment map shows you the current status of each productive configuration of all processes for which you are a team member.
+The deployment map shows the current status of each run configuration of all processes for which you are a team member.
```

Comment:
"of each" and "of all" and multiple prepositional phrases make this sentence hard to process. Would one of the following work?

Is it important to say "processes for which you are a team member" or can you just say "your team's processes"?

The deployment map shows the current status of each run configuration for each of your team's processes.

or more generally (is it important to say "for each run configuration"?):

The deployment map shows the current run configuration statuses for each of your team's processes.

Reply:
I agree that the sentence is not easy to process, but feel that it is more precise, because:
Each process
- is contained in a project
- has its own team (only if the user belongs to this team, he can view the process project)
- has a set of run configurations

Reply:
I agree with Beth that this sentence is hard to read. Maybe you can split this sentence in two:
The deployment map shows the current status of the running configurations. The running configurations belong to the running processes you are a team member of.
This is just an idea. I don't like my sentence hahahaha

Reply:
Yeah! Your option sounds great! Go ahead!